Curs11/C/01-copy_text.c

```diff
@@ -0,0 +1,81 @@
+/* Programarea Calculatoarelor CA, 2022-2023
+*
+*	Program care realizeaza copierea unui fisier text.
+*
+*/
+
+#include <stdio.h>
+
+#define FILENAME_LENGTH 101
+
+/// <summary>
+/// Copies the contents from <paramref name="source" /> to <paramref name="dest" />.
```

Comment:
We don't do HTML here. 😅 

Maybe you can get inspired by Linux command line syntax from man.  [1]

[1] [man/cp](https://man7.org/linux/man-pages/man1/cp.1.html)

Reply:
Sorry, this is how things are documented in Visual Studio. I will change it.